File: src/ru/luk/commands/EditableStands.java
package ru.luk.commands;

import org.bukkit.ChatColor;
import org.bukkit.entity.Player;
import org.bukkit.command.*;
import ru.luk.handlers.ArmorStand;

import java.util.ArrayList;
import java.util.List;


public class EditableStands implements CommandExecutor, TabExecutor {

    @Override public boolean onCommand(CommandSender sender, Command command, String s, String[] args) {
        if (args.length == 0) {
            return false;
        }

        if (args[0].equals("reload")) {
            ru.luk.EditableStands.plugin.reloadConfig();

            ArmorStand.playSounds = ru.luk.EditableStands.plugin.getConfig().getBoolean("playSounds");
            ArmorStand.removeItems = ru.luk.EditableStands.plugin.getConfig().getBoolean("withdrawItems");
            ArmorStand.removeItemsInCreative = ru.luk.EditableStands.plugin.getConfig().getBoolean("withdrawItemsInCreative");
            ArmorStand.damageTools = ru.luk.EditableStands.plugin.getConfig().getBoolean("damageTools");
            ArmorStand.damageToolsInCreative = ru.luk.EditableStands.plugin.getConfig().getBoolean("damageToolsInCreative");
            ArmorStand.dropItems = ru.luk.EditableStands.plugin.getConfig().getBoolean("dropItems");
            ArmorStand.armsAdd = ru.luk.EditableStands.plugin.getConfig().getBoolean("armsAdd");
            ArmorStand.armsRemove = ru.luk.EditableStands.plugin.getConfig().getBoolean("armsRemove");
            ArmorStand.plateAdd = ru.luk.EditableStands.plugin.getConfig().getBoolean("plateAdd");
            ArmorStand.plateRemove = ru.luk.EditableStands.plugin.getConfig().getBoolean("plateRemove");
            ArmorStand.doSmall = ru.luk.EditableStands.plugin.getConfig().getBoolean("doSmall");
            ArmorStand.doBig = ru.luk.EditableStands.plugin.getConfig().getBoolean("doBig");

            sender.sendMessage(ChatColor.GREEN + "Plugin successfully reloaded!");

        } else if (args[0].equals("info")) {
            sender.sendMessage(String.format("%sAuthor %s> %s_LuK__\n%sVersion %s> %s%s", ChatColor.GRAY, ChatColor.DARK_GRAY, ChatColor.WHITE,
                                                                                            ChatColor.GRAY, ChatColor.DARK_GRAY, ChatColor.WHITE, "1.0"));
        }

        return true;
    }

    @Override public List<String> onTabComplete(CommandSender sender, Command command, String s, String[] args) {
        if (sender instanceof Player) {
            List<String> arguments = new ArrayList<>();

            if (args.length == 1) {
                arguments.add("info");
                arguments.add("reload");
            }

            return arguments;
        }
        return null;
    }
}

File: src/ru/luk/commands/EditableStands.kt
package ru.luk.commands
import ru.luk.EditableStands.Companion.plugin
import ru.luk.handlers.ArmorStand;

import org.bukkit.ChatColor;
import org.bukkit.entity.Player;
import org.bukkit.command.*;


class EditableStands : CommandExecutor, TabExecutor {
    override fun onCommand(sender: CommandSender, command: Command, s: String, args: Array<String>): Boolean {
        if (args.isEmpty()) return false

        when(args[0]) {
            "reload" -> {
                plugin.reloadConfig()
                ArmorStand.playSounds = plugin.config.getBoolean("playSounds")
                ArmorStand.removeItems = plugin.config.getBoolean("removeItems")
                ArmorStand.removeItemsInCreative = plugin.config.getBoolean("removeItemsInCreative")
                ArmorStand.damageTools = plugin.config.getBoolean("damageTools")
                ArmorStand.damageToolsInCreative = plugin.config.getBoolean("damageToolsInCreative")
                ArmorStand.dropItems = plugin.config.getBoolean("dropItems")
                ArmorStand.armsAdd = plugin.config.getBoolean("armsAdd")
                ArmorStand.armsRemove = plugin.config.getBoolean("armsRemove")
                ArmorStand.plateAdd = plugin.config.getBoolean("plateAdd")
                ArmorStand.plateRemove = plugin.config.getBoolean("plateRemove")
                ArmorStand.doSmall = plugin.config.getBoolean("doSmall")
                ArmorStand.doBig = plugin.config.getBoolean("doBig")

                sender.sendMessage(ChatColor.GREEN.toString() + "Plugin successfully reloaded!")
            }

            "info" -> {
                sender.sendMessage("${ChatColor.GRAY}Author ${ChatColor.DARK_GRAY}> ${ChatColor.WHITE}_LuK__\n${ChatColor.GRAY}Version ${ChatColor.DARK_GRAY}> ${ChatColor.WHITE}1.0")
            }
        }
        return true
    }

    override fun onTabComplete(sender: CommandSender, command: Command, s: String, args: Array<String>): MutableList<String>? {
        if (sender is Player) {
            return if (args.size == 1) mutableListOf("info", "reload") else mutableListOf()
        }
        return null
    }
}